bin/node/runtime/src/lib.rs

```diff
@@ -237,7 +236,6 @@ parameter_types! {
 	pub const DepositBase: Balance = deposit(1, 88);
 	// Additional storage item size of 32 bytes.
 	pub const DepositFactor: Balance = deposit(0, 32);
```

Comment:
isn't is possible to use `ConstU128<{ deposit(0, 32) }` instead of this ?
I think we should also get rid of them then.